.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: xenial
+
+language: ruby
+
+git:
+  depth: 200
+
+rvm:
+  - 2.3.3
+
+before_install:
+  - sudo apt-get install -y libpango1.0-dev ghostscript
+  - gem install asciidoctor -v 1.5.8
+  - gem install coderay json-schema
+  - gem install ttfunk -v 1.5.1
+  - gem install asciidoctor-pdf -v 1.5.0.beta.8 --pre
+  - MATHEMATICAL_SKIP_STRDUP=1 gem install asciidoctor-mathematical -v 0.2.2
+
+script:
+  - git describe --tags --dirty
+  - make api c env ext
```

Comment:
Should we add C++ for OpenCL there as well?

Reply:
Yes, I'll add this.  Actually, I think we can just run `make`, and pick up everything by default?

Reply:
I looked at some options here:

* Just running `make` doesn't do anything, so that's not an option.
* Running `make all` builds more, but also doesn't include C++ for OpenCL.

I'm inclined to leave this as-is for now, and we can always update the build targets for the CI builds later.

Reply:
Yes, let's keep what you have, it's better than nothing and easy to build on top of.